app/models/facility.rb

```diff
@@ -73,6 +73,10 @@ def destroy
     # super
   end
 
+  def cross_billing?
```

Comment:
We have a `cross_facility?` and `single_facility?` method we should be able to use.